composer.json

```diff
@@ -25,6 +25,9 @@
         "ocramius/proxy-manager": "~0.5"
     },
     "require-dev": {
+        "ext-mongo" : "*",
```

Comment:
`ext-mongo` is required by `doctrine/mongodb-odm`, we do not need to require it ourselves.